utils/utils.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2023 CloudWeGo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+import (
+	"io"
+	"os"
+)
+
+func ReadFileAll(filepath string) ([]byte, error) {
+	file, err := os.Open(filepath)
+	if err != nil {
+		return nil, err
+	}
+
+	defer file.Close()
+
+	data, err := io.ReadAll(file)
+	if err != nil {
+		return nil, err
+	}
+
+	return data, nil
+}
+
+func PathExists(path string) (bool, error) {
```

Comment:
Why not use `os.IsNotExist` directly? Is the `os.Stat` necessary here?

Reply:
`os.IsNotExist` can only determine whether the error is due to a file not existing. Its input parameter is of type "error" and cannot directly take a file path as input. We still need to use `os.Stat` to obtain file information. 

However, I noticed that it is recommended to use `errors.Is()` for this purpose in newer versions, and it has been corrected accordingly.